LongWarOfTheChosen/Localization/LW_PerkPack_Integrated/XComGame.int

```diff
@@ -1061,10 +1061,10 @@ LocPromotionPopupText="<Bullet/> Your explosive attacks can inflict critical dam
 
 [RapidFire X2AbilityTemplate]
 LocFriendlyName="Rapid Fire"
-LocLongDescription="Fire twice in a row at an enemy. Each shot suffers an Aim penalty of <Ability:RAPIDFIREPENALTY/>. There is no cooldown on this ability."
+LocLongDescription="Fire twice in a row at an enemy. Each shot suffers an Aim penalty of <Ability:RAPIDFIREPENALTY/>. Can be used once per turn."
```

Comment:
`LocHelpText` should also have the additional text.

Reply:
It probably should. However, there are a number of discrepancies between LocHelpText and LocLongDescription (and LocLongDescription) in various descriptions: often, one describe one aspect while the other does not. I couldn't sense a pattern of what should contain what information, so I didn't make a point of adding information to LocHelpText LocLongDescription lines. For this particular case, sometimes information about cooldown is added to LocHelpText, but very often it doesn't.